### ISSUE TITLE
[BUG] cosize behaves inconsistently.

### ISSUE THREAD

**Issue by eatingtomatoes** [NVIDIA/cutlass#1178]
**Describe the bug**

 The behavior of cosize written in Python is inconsistent with that written in C++

C++
```c++
auto layout = make_layout(make_shape(_2 {}, _4 {}), make_stride(_2 {}, _4 {}));
print(cosize(layout)); // => _15
```
Python
```python
>>> cosize(Layout((2, 4), (2, 4)))
16
```

**Comment by thakkarV** [comment]
@ccecka

**Comment by ccecka** [comment]
Thanks, this is a bug in pycute!

**Comment by eatingtomatoes** [comment]
Should I close this issue now or wait for a fix?

**Comment by mnicely** [comment]
@eatingtomatoes I'll close once we have know the fix version

**Comment by mnicely** [comment]
Resolved with v3.4 release
